style.css

```diff
@@ -0,0 +1,11 @@
+header{
+    position: fixed;
+    width: 100%;
+    top: 0;
+    left: 0;
+}
+@media only screen and (max-width: 600px){
+    body { background-color: hotpink;
+
+    }
```

Comment:
I suggest that you move the closing brace back to the same line as the "body" selector for cleaner code.